src/DropDown/DropDown.styles.ts

```diff
@@ -65,7 +67,8 @@ export const StyledListItem = styled.li<StyleProps>`
   margin: 0;
   padding: ${p => p.theme.spacing.xsmall};
   color: ${p => p.theme.colors.gray4};
-  background: ${p => (p.highlighted ? p.theme.colors.gray3 : 'transparent')};
+  background: ${p => p.highlighted && p.theme.colors.gray1};
+  background: ${p => p.selectedItem && p.theme.colors.gray3};
```

Comment:
I _think_ this might throw console errors when pulled into maps. I don't think it likes `true` as a value being pased to the DOM. I might be wrong though

Reply:
I tried it and it didn't, but it is probably a bit ambiguous. I'll use the styled-components `css` import instead.